src/scripts/main.js

```diff
@@ -1,3 +1,40 @@
 'use strict';
 
-// write code here
+const people = document.querySelectorAll('li');
+const list = document.querySelector('ul');
+
+function change(salary) {
+  return +salary.slice(1).replaceAll(',', '');
+};
+
+const sortList = listToSort => {
+  const currentList = Array.prototype.slice.call(listToSort, 0);
+
+  currentList
+    .sort((a, b) => change(b.dataset.salary) - change(a.dataset.salary))
+    .map(x => list.append(x));
```

Comment:
avoid name like a,b,x,i and other letters, create meaningful names. item, prevItem, curItem, employee, person, element and so on

Reply:
map is useless here, you do not use a returned array, so just use forEach

Reply:
Done